sanic/policy.py

```diff
@@ -0,0 +1,13 @@
+import uvloop
+
+_policy = uvloop.EventLoopPolicy()
+
+def get_event_loop():
```

Comment:
It could possibly be good, maybe each app has it's own instantiated event policy?